pom.xml

```diff
@@ -0,0 +1,145 @@
+    <version>0.93.1.0</version>
+
+    <licenses>
+        <license>
+            <name>CC BY-NC-ND 3.0</name>
+            <url>http://creativecommons.org/licenses/by-nc-nd/3.0/</url>
+            <comments>Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported</comments>
+        </license>
+    </licenses>
+
+    <organization>
+        <name>TownyAdvanced</name>
+        <url>https://github.com/TownyAdvanced</url>
+    </organization>
+
+    <properties>
+        <project.bukkitAPI>1.13</project.bukkitAPI>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <!-- For use with GitHub Package Registry -->
+    <!-- Requires repo write access, and proper local maven configuration -->
+    <!-- https://help.github.com/en/articles/about-github-package-registry -->
+    <distributionManagement>
+        <repository>
+            <id>github</id>
+            <name>GitHub TownyAdvanced Apache Maven Packages</name>
+            <url>https://maven.pkg.github.com/TownyAdvanced</url>
+        </repository>
+    </distributionManagement>
+    <scm>
+        <connection>scm:git:https://github.com/TownyAdvanced/Towny.git</connection>
+        <developerConnection>scm:git:https://github.com/TownyAdvanced/Towny.git</developerConnection>
+        <url>https://github.com/TownyAdvanced/Towny</url>
+    </scm>
+
+    <repositories>
+        <repository>
+            <id>spigot-repo</id>
+            <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
+        </repository>
+        <repository>
+            <id>creatorfromhell-repo</id>
+            <url>https://dl.bintray.com/theneweconomy/java/</url>
+        </repository>
+        <repository>
+            <id>vault-repo</id>
+            <url>http://nexus.hc.to/content/repositories/pub_releases</url>
+        </repository>
+        <repository>
+            <id>ess-repo</id>
+            <url>https://ci.ender.zone/plugin/repository/everything/</url>
+        </repository>
+        <repository>
+            <id>groupmanager-repo</id>
+            <url>https://raw.githubusercontent.com/GroupManager/GroupManager/maven/</url>
+            <!-- Not browsable from this URL (HTTP 400) - Works fine as Repo, though. For Ver. # - see https://github.com/GroupManager/GroupManager/ -->
+        </repository>
+        <repository>
+            <id>citizens-repo</id>
+            <url>http://repo.citizensnpcs.co/</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.bukkit</groupId>
+            <artifactId>bukkit</artifactId>
+            <version>1.13.2-R0.1-SNAPSHOT</version>
+            <scope>provided</scope>
+            <!-- NOTE: Make sure this version works with project.bukkitAPI set above! -->
+        </dependency>
+        <dependency>
+            <groupId>net.tnemc</groupId>
+            <artifactId>Reserve</artifactId>
+            <version>0.1.0.12</version>
```

Comment:
This needs updated to version 0.1.2.0, as this is the version that plugins such as CMI, Ultimate Economy, etc will be implementing on due to the addition of the async-compatible API.

Reply:
Thanks for pointing it out!